src/main/java/net/szum123321/textile_backup/core/restore/RestoreBackupRunnable.java

```diff
@@ -63,7 +63,7 @@ public void run() {
 
         try {
             tmp = Files.createTempDirectory(
-                    ctx.server().getRunDirectory().toPath(),
+                    ctx.server().getRunDirectory().toAbsolutePath(),
```

Comment:
Not sure i understand the intention here?

Reply:
it said toPath wasn't found when i compiled.